.github/workflows/python-linters.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
Not so important, but Python 3.9 support will end very soon (in the next weeks). We could already test with 3.10 to 3.12 (or even 3.13).